S3 trick documentation (#6540)
randovania/randovania: randovania/games/fusion/logic_database/Sector 3 PYR.txt

      Any of the following:
          # Get past wall enemy. For simplicity's sake, we assume the enemies are always there.
          Charge Beam or Missiles ≥ 5 or Morph Ball or Screw Attack or Wave Beam or Normal Damage ≥ 80

>           # Shinespark https://youtu.be/3sm2XhRsRRI

Reviewer: ```suggestion
          # Shinespark: https://youtu.be/3sm2XhRsRRI
```

          Morph Ball and Screw Attack and Knowledge (Beginner)
          All of the following:
              Can Use Bombs
              Any of the following:
                  Can Use Springball

>                   # https://youtu.be/UBvgvEpVc-Q

Reviewer: ```suggestion
                  # Mid Air Morph: https://youtu.be/UBvgvEpVc-Q
```

Author: kinda seems redundant to put mid-air morph as a comment for a trick that is called mid-air morph but I don't care enough to argue

  > Door to Deserted Runway
      Any of the following:
          Space Jump
          All of the following:

>               # Stand on Frozen Enemy and Walljump off
>               Stand On Frozen Enemies (Beginner)
>               Any of the following:
>                   # https://youtu.be/ZU70fb8aivM
>                   Hi-Jump
>                   # https://youtu.be/s6HpLtE82Ug
>                   Wall Jump (Beginner)

Reviewer: ```suggestion
              # Stand on Frozen Enemy and Jump off of it
              Stand On Frozen Enemies (Beginner)
              Any of the following:
                  # With HJ: https://youtu.be/ZU70fb8aivM
                  Hi-Jump
                  # With WJ: https://youtu.be/s6HpLtE82Ug
                  Wall Jump (Beginner)
```

          Space Jump
          All of the following:
              Can Freeze Enemies

>               Any of the following:
>                   Hi-Jump
>                   # https://youtu.be/vxQS1_ohmS4
>                   Wall Jump (Beginner)

Reviewer: change comment at top to sth like "freeze enemy and jump off of it"
```suggestion
              Any of the following:
                  Hi-Jump
                  # With WJ: https://youtu.be/vxQS1_ohmS4
                  Wall Jump (Beginner)
```

Reviewer: This wasnt addressed

      Trivial
  > Door to Sova Processing (Lower)
      Can Break Single Bomb Blocks
  > Door to Sova Processing (Upper)
      All of the following:

>           # https://youtu.be/0ILTTSV-Uc4

Reviewer: ```suggestion
          # Shinespark from next room: https://youtu.be/0ILTTSV-Uc4
```
